webpack/src/Components/EmptyState/Constants.js

```diff
@@ -0,0 +1,2 @@
+export const ADD_CONTENT = '[PLUGIN_TEMPLATE] ADD_CONTENT';
```

Comment:
will the name be replaced with a script?
after it will be replaced we should keep on the underline convention.

Reply:
It is written in a way to make it clear for renaming, in redux console it would bold enough like that IMHO.

Reply:
yes, but can it be done automatically with the `rename.rb`?
can you also rename this file into `EmptyStateConstants.js`

Reply:
what do you think @amirfefer ?